### PR TITLE
Add info page

### PR DESCRIPTION
**Pull request by Lev-Stambler**
- Add an info page describing the project
- Clean up CSS

[The site](https://schopal.neocities.org)

### CODE REVIEW

**Review by vitashubin — APPROVED** [review]
Love the addition of this page, super helpful and the UI looks great!

File: apps/api-frontend/src/app/app.css [hunk 1]
```diff
@@ -1,14 +1,20 @@
-/*
- * Remove template code below
- */
-
 .app {
-  font-family: sans-serif;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```
Comment (cwlarson-google) [inline apps/api-frontend/src/app/app.css:2]: This isn't a problem per se, but you don't need to provide so many font alternatives.  Defaulting to sans-serif is probably enough

File: apps/api-frontend/src/app/app.css [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
   min-width: 300px;
   max-width: 600px;
   margin: 50px auto;
 }
 
+.app a {
+  text-decoration: none;
+  color: blue;
```
Comment (cwlarson-google) [inline apps/api-frontend/src/app/app.css:10]: Minor nit: `color: blue` (#0000ff) is a pretty harsh color and is usually avoided on websites (when there's a designer involved, at least).  Other shades of blue stand out a bit more and aren't as hard on the eyes

I see GitHub using #005cc5 on this page (for CSS selectors), maybe try that out

File: apps/api-frontend/src/app/pages/info-page/info.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './info.css';
+
+export default function InfoPage() {
+  return (
+    <div className="info">
+      <h3>Info about Schopal</h3>
+      <p>
+        Schopal is a search engine for Scholarly articles. Using open source APIs, Schopal finds scholarly articles, and then analyzes them according to your search.
+        Schopal will then return results <div className="bold">by paragraph</div> rather than by article. Thus, you may see the same article appear twice.
```
Comment (cwlarson-google) [inline apps/api-frontend/src/app/pages/info-page/info.tsx:10]: Why not use `<b>text</b>` instead of `<div className="bold">text</div>`?

**Review by cwlarson-google — APPROVED** [review]

File: apps/api-frontend/src/app/pages/info-page/info.tsx (hunk 3 above)
Comment (vitashubin) [reply apps/api-frontend/src/app/pages/info-page/info.tsx:10]: +1 good catch Casey

**Pull request merged**